tutorials/multiple_robots.md

```diff
@@ -110,11 +131,11 @@ Once the Create® 3 application has restarted, try calling `ros2 topic list` on
 
 ## Namespacing
 
-Namespacing is a method of adding a prefix to topic names to group certain topics, or to make them unique. 
+Namespacing is a method of adding a prefix to topic names to group certain topics, or to make them unique.
 By namespacing all topics on a TurtleBot 4, we can have a full set of unique topics for that robot. This allows
 us to run multiple robots on the same `ROS_DOMAIN_ID`.
 
-It is common to use the name of the robot as the namespace. For example, 
+It is common to use the name of the robot as the namespace. For example,
```

Comment:
Superfluous new line (flagged because the space was removed as part of that other discussion)